### PR TITLE
Bug domain favicon

### PR DESCRIPTION
**Pull request by rishi9684**
#favicon icon added as per domain
#title added as domain

### CODE REVIEW

File: layouts/admin.vue [hunk 1]
```diff
@@ -488,10 +492,26 @@ export default {
         this.$apolloHelpers.onLogout()
       }
     },
+    getDomain() {
+      if (process.client) {
```
Comment (SukantGujar) [inline layouts/admin.vue:496]: Test if this works locally, when the hostname would just be localhost.

File: layouts/default.vue [hunk 2]
```diff
@@ -438,10 +442,26 @@ export default {
         this.$apolloHelpers.onLogout()
       }
     },
+    getDomain() {
```
Comment (SukantGujar) [inline layouts/default.vue:445]: This code is being repeated several times, create a mixin.

**Review by SukantGujar — CHANGES_REQUESTED** [review]

File: layouts/admin.vue (hunk 1 above)
Comment (rishi9684) [reply layouts/admin.vue]: Yes its working fine for localhost, its display title as localhost

File: layouts/default.vue (hunk 2 above)
Comment (rishi9684) [reply layouts/default.vue]: created mixin for getDomain()

**Review by SukantGujar — APPROVED** [review]

File: layouts/admin.vue (hunk 1 above)
Comment (rishi9684) [reply layouts/admin.vue]: image

**Pull request merged**